README.md

```diff
@@ -114,8 +114,10 @@ the offset and length of some Wasm memory, to allow for safe access between the
 caller and the contract:
 
 ```rust
-/// Refers to some heap allocated data in wasm.
-/// A pointer to this can be returned over ffi boundaries.
+/// Refers to some heap allocated data in Wasm.
+/// A pointer to an instance of this can be returned over FFI boundaries.
+///
+/// This struct is crate internal since the VM defined the same type independently.
```

Comment:
You can use `pub(crate) struct Region` there.
It allows other mods in the crate to import it freely, but doesn't expose externally

Reply:
Jap, I came across that notation.

But since it is not exposed in `lib.rs`, we get the same result right now. I had to conditionally import `memory.rs` because otherwise Rust complains about dead code in the IDE and non-Wasm builds.